client/src/pages/Signup.js

```diff
@@ -0,0 +1,138 @@
+import { useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+function Signup() {
+    const emailRegEx = /^[A-Za-z0-9]([-_.]?[A-Za-z0-9])*@[A-Za-z0-9]([-_.]?[A-Za-z0-9])*\.[A-Za-z]{2,3}$/;
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [nickname, setNickname] = useState('');
+    const [gender, setGender] = useState('');
+    const [location, setLocation] = useState('시/도');
+    const [subLocation, setSubLocation] = useState('');
+    const [checkbox1, setCheckbox1] = useState(false);
+    const [checkbox2, setCheckbox2] = useState(false);
+    const navigate = useNavigate();
+
+    const koreaLocation = {
```

Comment:
@sungminoh Doesn't need to relocate koreaLocation to config file?

Reply:
[c3b52e1](https://github.com/tmznwnel00/date_dive/pull/6/commits/c3b52e1a318941544f1e93fcc8286c0d31609dc3)